src/tests/Microsoft.Diagnostics.NETCore.Client/CommonHelper.cs

```diff
@@ -18,12 +19,17 @@ public class CommonHelper
         public static string GetTraceePath(string traceeName = "Tracee", string targetFramework = "netcoreapp3.1")
         {
             var curPath = Directory.GetCurrentDirectory();
-;
+
             var traceePath = curPath
                 .Replace(System.Reflection.Assembly.GetCallingAssembly().GetName().Name, traceeName)
                 .Replace("netcoreapp3.1", targetFramework);
 
-            return Path.Combine(traceePath, Path.ChangeExtension(traceeName, ".dll"));
+            traceePath = Path.Combine(traceePath, Path.ChangeExtension(traceeName, ".dll"));
```

Comment:
Can we rename the function and variable to get tracee with args? It's no longer just the path